README.md

```diff
@@ -52,24 +52,28 @@ Trader is an autonomous service that performs bets on existing prediction market
     export SAFE_CONTRACT_ADDRESS="YOUR_SAFE_ADDRESS"
     export OMEN_CREATORS='["CREATOR_0", "CREATOR_1", ...]'
     
-    # Optional
+    # Optional. The following example values bet a variable amount depending on the
+    # prediction confidence. Here, amounts vary between 0.005 xDAI (60% confidence)
+    # and 0.1 xDAI (100% confidence). Please, adjust these values accordingly.
     export BET_AMOUNT_PER_THRESHOLD_000=0
     export BET_AMOUNT_PER_THRESHOLD_000=0
     export BET_AMOUNT_PER_THRESHOLD_010=0
     export BET_AMOUNT_PER_THRESHOLD_020=0
     export BET_AMOUNT_PER_THRESHOLD_030=0
     export BET_AMOUNT_PER_THRESHOLD_040=0
     export BET_AMOUNT_PER_THRESHOLD_050=0
-    export BET_AMOUNT_PER_THRESHOLD_060=600000000000000000
-    export BET_AMOUNT_PER_THRESHOLD_070=900000000000000000
-    export BET_AMOUNT_PER_THRESHOLD_080=1000000000000000000
-    export BET_AMOUNT_PER_THRESHOLD_090=10000000000000000000
-    export BET_AMOUNT_PER_THRESHOLD_100=100000000000000000000
-    export BET_THRESHOLD=100000000000000000
+    export BET_AMOUNT_PER_THRESHOLD_060=5000000000000000
+    export BET_AMOUNT_PER_THRESHOLD_070=7000000000000000
+    export BET_AMOUNT_PER_THRESHOLD_080=10000000000000000
+    export BET_AMOUNT_PER_THRESHOLD_090=50000000000000000
+    export BET_AMOUNT_PER_THRESHOLD_100=100000000000000000
+
+    # Threshold for placing a bet
+    export BET_THRESHOLD=2000000000000000
```

Comment:
No, because these values satisfy 

min_bet_amount - (max)fees > BET_THRESHOLD

5000000000000000 - 900000000000000 = 4100000000000000 > 2000000000000000 (wei)
**min_bet_amount - (max)fees = 0.005 - 0.0009 = 0.0041 > 0.002 = BET_THRESHOLD**

Reply:
The fee [is usually](https://api.thegraph.com/subgraphs/name/protofire/omen-xdai/graphql?query=%7B%0A++fixedProductMarketMakers%28%0A++++where%3A+%7B%0A++++++creator_in%3A+%5B%220x89c5cc945dd550BcFfb72Fe42BfF002429F46Fec%22%5D%2C%0A++++++outcomeSlotCount%3A+2%2C%0A++++++isPendingArbitration%3A+false%0A++++%7D%2C%0A++++orderBy%3A+creationTimestamp%0A++++orderDirection%3A+desc%0A++%29%7B%0A++++fee%0A++%7D%0A%7D) `20_000_000_000_000_000`.